packages/debug/src/browser/debug-prefix-configuration.ts

```diff
@@ -49,15 +54,34 @@ export class DebugPrefixConfiguration implements CommandContribution, CommandHan
     @inject(LabelProvider)
     protected readonly labelProvider: LabelProvider;
 
+    @inject(StatusBar)
+    protected readonly statusBar: StatusBar;
+
     readonly prefix = 'debug ';
     readonly description = 'Debug Configuration';
+    readonly statusBarId = 'select-run-debug-statusbar-item';
 
     private readonly command: Command = {
         id: 'select.debug.configuration',
         category: 'Debug',
         label: 'Select and Start Debugging'
     };
 
+    @postConstruct()
+    protected update(): void {
```

Comment:
`init` is better name, since it happens only once on the construction

Reply:
@akosyakov unfortunately I cannot use `protected init` as it is present in the `QuickOpenHandler` for which we implement:

https://github.com/eclipse-theia/theia/blob/5818d507201f84701f99788c76a465028bca59b0/packages/core/src/browser/quick-open/prefix-quick-open-service.ts#L50-L55

Reply:
`initialize`? :D

Reply:
> `initialize`? :D

haha sure :)